README.md

```diff
@@ -1,155 +1,111 @@
 
-## 2. capa-controller-role
-Please read the [README.md](./capa-controller-role/README.md) of the capa-controller-role module.
+You will be asked for the following parameters:
 
+- `AdminRoleName`: The name of the role that will be created. Default is `GiantSwarmAdmin`. You dont need to change this unless you have a specific requirement.
 
-# Vintage
-## Before starting
+Review the changes and click `Create stack`. In case of any error, please check the `Events` tab in the CloudFormation console and report the error to the Giant Swarm staff.
 
-Make sure to adjust AWS account limits according to [these
-docs](https://docs.giantswarm.io/guides/prepare-aws-account-for-tenant-clusters/#limits).
+### Terraform
 
-For Vintage take a look at theese three modules in this repository:
+#### Requirements
 
-1. [admin-role](./admin-role) which creates a role and a policy for our
-   staff to be able to operate the infrastructure created by our automation in
-   case of failures.
-2. [aws-operator-user](./aws-operator-user) which creates a user and its policy
-   to be used for our automation to manage the infrastructure.
-3. [aws-operator-role](./aws-operator-role) which creates
-   the role and policy to be assumed for the automation user to create and manage
-   resources.
+- `terraform` installed
+- working AWS credentials set to the desired target account
+- AWS region has to be set  either via aws profile or via env `AWS_REGION`
 
-## 1. admin-role
+### Adjust variables
 
-For all AWS accounts part of the platform, does not matter if they are for
-control plane or tenant clusters, we need to have access in order to debug and
-operator the infrastructure. To do so, please run this module in the target
-account:
+- `principal_arns_giantswarm_root_account` - can be adjusted to be more strict and specify user which will assume the role. Default is `arn:aws:iam::084190472784:root`. You dent need to change this unless you have a specific requirement.
+- `admin_role_name` - can be adjusted to be more strict and specify role name. You dent need to change this unless you have a specific requirement.
 
-```hcl
-module "giantswarm-cp-prereqs" {
-  source = "git@github.com:giantswarm/giantswarm-aws-account-prerequisites//admin-role"
-}
+### Execution
 
-output "giantswarm-admin-role" {
-  value = "${module.giantswarm-cp-prereqs.giantswarm-admin-role}"
-}
 ```
-
-The created role and policy name is `GiantSwarmAdmin`.
+terraform init .
+terraform apply -var="admin_role_name=GiantSwarmAdmin
+```
 
 The created role ARN needs to be supplied to Giant Swarm.
 
-## 2. aws-operator-user
+## 2. capa-controller-role
```

Comment:
I have decided to bring all explanation here since now vintage is gone and IMO is simple